target_iceberg/sinks.py

```diff
@@ -20,27 +24,33 @@ def __init__(self, target, schema, stream_name, key_properties) -> None:
             stream_name=stream_name,
             key_properties=key_properties,
         )
-        self.table_name = self.config.get("table_name")
-        self.flatten_max_level = self.config.get("max_flatten_level", 100)
-
-        # Extra fields
-        self.extra_values = (
-            dict([kv.split("=") for kv in self.config["extra_fields"].split(",")])
-            if self.config.get("extra_fields")
-            else {}
-        )
-        self.extra_values_types = {}
-        if self.config.get("extra_fields_types"):
-            for field_type in self.config["extra_fields_types"].split(","):
-                field_name, _type = field_type.split("=")
-                self.extra_values_types[field_name] = {"type": [_type]}
+        table_name_prefix = f"{self.config.get('table_name_prefix')}_" if self.config.get("table_name_prefix") else ""
+        self.table_name = f"{self.config['db_name']}.raw_{table_name_prefix}{self.__class__.to_snake_case(self.stream_name)}"
```

Comment:
About the table name, maybe we can use a `prod` config (boolean) that if not set the table will be created in scratch. And maybe the user can decide if the `raw_` will be included in the name.
```suggestion
        self.table_name = f"{self.config['db_name'] if self.config.get('prod') else 'scratch'}.{table_name_prefix}{self.__class__.to_snake_case(self.stream_name)}"
```